### PR TITLE
TABLE SCHEMA

### PR DESCRIPTION
**Pull request by emmanuellmhango**
Created a schema.sql file with tables that are analogical to the structure of the classes that i have created:

- music_albums table (added all properties and associations from the parent Item class as table columns)
- genres table

### CODE REVIEW

File: schema/schema.sql [hunk 1]
```diff
@@ -0,0 +1,29 @@
+-- CREATE MUSIC_ALBUM TABLE
+CREATE TABLE music_album (
+  id SERIAL PRIMARY KEY,
+  author INTEGER,
+  genre INTEGER,
+  source INTEGER,
+  label INTEGERL,
```
Comment (mperezbazan) [inline schema/schema.sql:7]: - [] Kindly review the typo in the INTEGER word

File: schema/schema.sql [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  FOREIGN KEY (author) REFERENCES author(id),
+  FOREIGN KEY (source) REFERENCES source(id),
+  FOREIGN KEY (label) REFERENCES label(id),
+  FOREIGN KEY (genre) REFERENCES genre(id)
+);
+
+-- CREATE GENRE TABLE
+CREATE TABLE genre (
+  id SERIAL PRIMARY KEY,
+  name VARCHAR(100) NOT NULL,
```
Comment (mperezbazan) [inline schema/schema.sql:20]: - [] Kindly review this line, the last column the comma is not needed

File: schema/schema.sql [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+);
+
+-- CREATE GENRE ITEMS TABLE
+CREATE TABLE genre_items (
+  id SERIAL PRIMARY KEY,
+  genre_id INTEGER NOT NULL
```
Comment (mperezbazan) [inline schema/schema.sql:26]: - [] Kindly review the comma at the end of the properties added, this is needed between the columns.

**Review by mperezbazan — CHANGES_REQUESTED** [review]
Hi @emmanuellmhango , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights :heavy_check_mark: 
- [ ] Git flow is used correctly :white_check_mark: 
- [ ] Descriptive pull request title and summary :white_check_mark: 

## Required Changes ♻️
- [ Required ] I've noticed that the social media links on your `readme` file are not working, please make sure that you provide links to `GitHub` `LinkedIn`, and `Twitter` for all the contributors of this project and that they are working. It's now a core requirement that all links provided on a readme file should not be dead links.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema/schema.sql [hunk 4]
```diff
@@ -4,7 +4,7 @@ CREATE TABLE music_album (
   author INTEGER,
   genre INTEGER,
   source INTEGER,
```
Comment (amexabee) [inline schema/schema.sql:6]: kindly, remove the reference to source there is no such a table in this schema.

**Review by amexabee — CHANGES_REQUESTED** [review]
# STATUS: Required Changes ♻️♻️♻️ 
## Hi @emmanuellmhango,  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close-gif-4

## Highlights

- Great implementation for the functions  :heavy_check_mark:👨🏻‍💻
- Great work using the right Gitflow.:heavy_check_mark:
- No Linter errors :heavy_check_mark:

## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: schema/schema.sql (hunk 4 above)
Comment (emmanuellmhango) [reply schema/schema.sql]: Dear @amexabee ,

Kindly check the revised code and advise if its okay now.

**Review by amexabee — APPROVED** [review]
## Hi @emmanuellmhango ,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### Congratulations! 🎉 
### Highlights
- Well organized and clean project ✔️
- Gitflow was applied well ✔️
- Good project structure ✔️
thanks-awesome

# Status: Approved ✔️✔️✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**